sdks/python/tox.ini

```diff
@@ -127,14 +127,14 @@ commands =
 [testenv:py3-lint]
 deps =
   pycodestyle==2.3.1
-  pylint==1.7.2
+  pylint==2.1.1
   future==0.16.0
   isort==4.2.15
   flake8==3.5.0
 setenv =
     BEAM_EXPERIMENTAL_PY3=1
 commands =
-  python --version
+  pylint --version
```

Comment:
Instead of removing "python --version", can you add "pylint --version" as a new line instead?

Reply:
Same